subsys/zigbee/lib/zigbee_helpers/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+zephyr_library()
+zephyr_library_sources_ifndef(CONFIG_ZB_TEST_MODE zigbee_helpers.c)
+
+zephyr_library_link_libraries(zboss)
```

Comment:
It looks like linking zboss library is required for other libraries to include ZBOSS headers.

Reply:
That's because in zboss and zigbee you use `target_include_directories()` instead of `zephyr_include_directories()`, hence the paths aren't available globally. IMO it would sense to make them global for zboss, but not necessiraly in this PR.